encoding/ccf/encode.go

```diff
@@ -61,7 +61,6 @@ func Encode(value cadence.Value) ([]byte, error) {
 	var w bytes.Buffer
 
 	enc := NewEncoder(&w)
-	defer enc.enc.Close()
```

Comment:
Was this removal intentional?

Reply:
Updated: 891935956430045008ec9cb4d3073931f23a1fc8